cmd/s3-peer-client.go

```diff
@@ -19,112 +19,66 @@ package cmd
 import (
 	"encoding/json"
 	"fmt"
-	"net/rpc"
 	"net/url"
 	"path"
 	"sync"
-	"time"
 )
 
 type s3Peers struct {
-	// A map of peer server address (in `host:port` format) to RPC
-	// client connections.
-	rpcClients map[string]*AuthRPCClient
+	// A map of peer server address (in `host:port` format) to
+	// BucketMetaState interface
+	BMSClients map[string]BucketMetaState
 
-	mutex *sync.RWMutex
-
-	// Is single-node?
-	isDistXL bool
+	// Is distributed?
+	IsDistXL bool
```

Comment:
This code is not going to turn in to a package in the future. So it is OK to keep it as lower case. Either way is not wrong, but we need to keep it consistent.